ladybug/common/pom.xml

```diff
@@ -12,6 +12,19 @@
 	<artifactId>frankframework-ladybug-common</artifactId>
 	<name>Frank!Framework Ladybug Commons</name>
 
+	<!-- Update the transitive CXF dependency by importing the cxf BOM of the correct version -->
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.apache.cxf</groupId>
+				<artifactId>cxf-bom</artifactId>
+				<version>4.0.6</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
```

Comment:
Won't it be easier to add it to the ladybug parent pom? That way you don't need to define it in the 3 ladybug modules.